be/src/storage/lake/gc.cpp

```diff
@@ -340,19 +357,20 @@ Status segment_gc(std::string_view root_location, TabletManager* tablet_mgr) {
         return iter_st;
     }
 
-    // Find orphan segments
-    ASSIGN_OR_RETURN(auto orphan_segments, find_orphan_segments(tablet_mgr, root_location, tablet_metadatas, txn_logs));
+    // Find orphan data files, include segment, del, and delvec
+    ASSIGN_OR_RETURN(auto orphan_datafiles,
+                     find_orphan_datafiles(tablet_mgr, root_location, tablet_metadatas, txn_logs));
     // Write orphan segment list file
     WritableFileOptions opts{
             .sync_on_close = false, .skip_fill_local_cache = true, .mode = FileSystem::CREATE_OR_OPEN_WITH_TRUNCATE};
     ASSIGN_OR_RETURN(auto orphan_list_file, fs->new_writable_file(opts, join_path(root_location, kGCFileName)));
-    RETURN_IF_ERROR(write_orphan_list_file(orphan_segments, orphan_list_file.get()));
-    // Delete orphan segment files
-    for (auto& seg : orphan_segments) {
-        LOG(INFO) << "Deleting orphan segment " << seg;
-        auto location = join_path(segment_root_location, seg);
+    RETURN_IF_ERROR(write_orphan_list_file(orphan_datafiles, orphan_list_file.get()));
+    // Delete orphan segment files and del files
+    for (auto& file : orphan_datafiles) {
+        LOG(INFO) << "Deleting orphan data files " << file;
```

Comment:
```suggestion
        LOG(INFO) << "Deleting orphan data file " << file;
```

Reply:
Done, and rename `segment_gc` to `datafile_gc`